src/v2/spec.yaml

```diff
@@ -1466,3 +1483,47 @@ paths:
                 $ref: "#/components/schemas/RelevantFollowersResponse"
         "400":
           $ref: "#/components/responses/400Response"
+  /farcaster/reactions:
+    get:
+      tags:
+        - Reaction
+      summary: Fetches reactions for a given user
+      description: Fetches reactions for a given user
+      operationId: user-reactions
+      externalDocs:
+        url: https://docs.neynar.com/reference/reactions
+      parameters:
+        - $ref: "#/components/parameters/ApiKey"
+        - name: fid
+          in: query
+          required: true
+          example: 3
+          schema:
+            $ref: "#/components/schemas/Fid"
+        - name: cursor
+          in: query
+          description: Pagination cursor.
+          schema:
+            type: string
+        - name: limit
+          in: query
+          description: Number of results to retrieve (default 25, max 100)
+          schema:
+            type: integer
+            default: 25
+            minimum: 1
+            maximum: 100
+        - reaction_type:
+          in: query
+          description: Type of reaction to fetch (like or recast or all)
+          schema:
+            $ref: "#/components/schemas/GetReactionTypes"
+      responses:
+        "200":
+          description: Successful operation.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/UserReactionsResponse"
+        "400":
```

Comment:
this will show up as `400ResponseReactionsApi` in the SDK. much better to have a standard object